src/components/AppHeader/index.js

```diff
@@ -114,8 +119,8 @@ class AppHeader extends Component {
       <header className={classes.base}>
         <Grid container spacing={0}>
           <Grid item container xs={2} lg={4}>
+            <FontAwesomeIcon className={classes.hamburger} icon={faBars} size="2x" onClick={this.toggleDrawer} />
```

Comment:
I would recommend just using the menu icon from material-ui rather than adding font awesome.

Thoughts?

Reply:
I definitely agree, forgot material-ui comes with those icons, good call.